templates/extras/calc.md

```diff
@@ -52,65 +30,60 @@ proof is complete. Use `_` as in the example above or `sorry` to stand
 for missing justifications. `sorry` will supress error messages
 entirely, while `_` will generate a guiding error message.
 
-If the structure of calc is incorrect (e.g., missing `:` or the
+If the structure of calc is incorrect (e.g., missing `:=` or the
 justification after it), you may see error messages that are obscure
-and/or red squiggles that end up under a random `...`. To avoid these,
+and/or red squiggles that end up under a random `_`. To avoid these,
 you might first populate a skeleton proof such as:
 
 ```lean
 example (A B C D : ℝ ) : A = D :=
-calc A = B : sorry
-...    = C : _
-...    = D : sorry
+calc A = B := sorry
+     _ = C := _
+     _ = D := sorry
 ```
 
 and then fill in the `sorry` and `_` gradually.
 
-In tactic mode calc should be terminated with a comma:
-```lean
-have H : A = D,
-{ calc A = B : sorry
-  ...    = C : sorry
-  ...    = D : _,
-  sorry },
-```
-and the `_` can be left in as they generate a subgoal to be resolved
```

Comment:
This sentence is still true (or at least, it's true with `?_` instead of `_`.

Reply:
Thanks will try this out soon.

Reply:
Okay added that information back in